### PR TITLE
[Data Factory] Add data flow compute property in IR.

### PR DESCRIPTION
**Pull request by jackmagic313**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json [hunk 1]
```diff
@@ -100,6 +100,10 @@
           "type": "integer",
           "minimum": 1
         },
+        "dataFlowProperties": {
```
Comment (ravbhatnagar) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json]: adding new writable properties is a breaking change and must be done in a new API version. But I think for datafactory an exception was given before. SO I am fine signing off on this. But please ensure customer cant unintentionally overwrite the goal state of the resource when making an update from an older SDK.

**Review by ravbhatnagar — COMMENTED** [review]
@jackmagic313 - few comments

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 2]
```diff
@@ -3751,6 +3752,7 @@
           }
         },
         "description": "Execute a data flow debug command.",
+        "x-ms-long-running-operation": true,
```
[the remark this answers is not in the document]
Comment (ravbhatnagar) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Jack Ma confirmed that this API POST action was added 2 days ago and this flag was missed. The SDKs have not been published, so shouldn't be breaking.

**Comment by ravbhatnagar** [comment]
Signing off from ARM side.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**